core/domain/summary_services.py

```diff
@@ -252,7 +253,7 @@ def get_top_rated_exploration_summary_dicts(language_codes):
 
```

Comment:
I don't think you need the tiebreak anymore. The Wilson score takes the number of ratings into account, so we can just tiebreak arbitrarily.